packages/components/atoms/f-link/src/components/_tests/Link.test.js

```diff
@@ -1,10 +1,163 @@
-import { shallowMount } from '@vue/test-utils';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
```

Comment:
Not sure if it's possible with CSS Modules (I know we've had issues with Jest + CSS Modules rendering out classnames previously), but would be good to have some tests for the props (so that it checks that classes are added when props are true (such as the bold/full-width ones).

Doesn't need to be part of this PR but would be good to see if this is testable or not in Jest with CSS Modules now.